File: LoopConstruction/Java/FibonacciSeries.java

import java.util.Scanner;

public class FibonacciSeries {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter the Nth Value ; ");
        int n = sc.nextInt();
        int first=0,second=1,next=0;
        for (int i = 0; i < n; i++) {
            if (i == 0) {
                System.out.print(first+" ");
            } else if (i == 1) {
                System.out.print(second+" ");
            } else {
                next = first + second;
                System.out.print(next+" ");
                first = second;
                second = next;
            }
        }
    }
}

File: LoopConstruction/C++/fibonacciSeries.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    long long first = 0, second = 1, next;

    cout << "Enter the number of terms in Fibonacci series: ";
    cin >> n;

    cout << "Fibonacci Series: ";

    for (int i = 0; i < n; i++) {
        if (i == 0) {
            cout << first << " ";
        } else if (i == 1) {
            cout << second << " ";
        } else {
            next = first + second;
            cout << next << " ";
            first = second;
            second = next;
        }
    }

    cout << endl;

    return 0;
}
